compiler/front/commands.nim

```diff
@@ -480,18 +462,22 @@ proc processCmdLine*(pass: TCmdLinePass, cmd: string; config: ConfigRef) =
         # only kept because of user expectations
         expectNoArg(cliFlagVersion, p.key, p.val)
         writeVersionInfo(config)
+        msgQuit(0)
```

Comment:
It's a small change (i.e., quitting the compiler processing a help or version flag) that doesn't affect much, but please list it in the PR message.